docs/data/toolpad/components/button.md

```diff
@@ -0,0 +1,55 @@
+# Button
+
+<p class="description">Learn about a button component and it's usage in Toolpad.</p>
+
+Buttons allow users to take actions, make choices with a single tap. Detailed documentation of the component properties is available in the reference section for [button](/toolpad/reference/components/button/#properties).
```

Comment:
You can list the use cases below here, something like: `The Button component allows you to take actions such as changing component state, calling a backend function or navigating to a page.`